server_ftp/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,7 @@
+To configure this module, you need to:
+
+#. Go ``to Settings/Technical/FTP/Servers``
+#. Find your favourite protocol and configure connection settings
+#. In the form view of the record, edit file structure.
+#. If more folders are needed, just add data records with a proper xml_id
```

Comment:
This is for telling users how to configure, so leave out the lines about data records.